build.sbt

```diff
@@ -243,8 +243,16 @@ lazy val play = project
   .settings(name := "caliban-play")
   .settings(commonSettings)
   .settings(
+    testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")),
     libraryDependencies ++= Seq(
-      "com.typesafe.play" %% "play" % playVersion
+      "dev.zio"                      %% "zio"                           % zioVersion,
```

Comment:
Some of these dependencies are obtained transitively from core, we don't need to specify them again, do we?